_api-reference/tasks.md

```diff
@@ -203,6 +219,8 @@ curl -XGET "localhost:9200/_tasks?actions=*search&detailed
 
 ```
 
+Note that `resource_stats` values will only be updated for tasks that support resource tracking. Moreover, `active_threads` refers to the number of threads currently working on the task while `thread_executions` refers to the number of threads that have been scheduled to work on the task, which includes both threads that have finished working on the task and threads currently working on the task.
+
```

Comment:
Can we add a table for defining all the stats emitted in `resource_stats` section. Like what does `min/max/average/total` mean here for a task similar to how you have defined `thread_info` section

Reply:
@sohami Thanks for the callout, added an additional table in the most recent revision.